api-specificatie/openapi.yaml

```diff
@@ -1130,32 +1165,25 @@ components:
       type: object
       properties:
         gebruiksdoel:
-          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
-            juistheid van de waarde in het corresponderende gegeven.
+          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
           type: boolean
         geometrie:
-          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
-            juistheid van de waarde in het corresponderende gegeven.
+          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
           type: boolean
         hoofdadresidentificatie:
-          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
-            juistheid van de waarde in het corresponderende gegeven.
+          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
           type: boolean
         nevenadresIdentificaties:
-          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
-            juistheid van de waarde in het corresponderende gegeven.
+          description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
           type: boolean
         pandIdentificaties:
```

Comment:
bij deze identificaties, bij adresseerbaarObjecten panden en hoofdadres, hebben we zowel de identificatie als de relatie in _links. Hoe maken we duidelijk dat ook de link mogelijk niet klopt, en evt. het embedded adres of pand?

Reply:
Daar zou ook een indicator opgenomen kunnen worden. Ik denk dat we hiervoor ook aan Pieter een voorstel moeten vragen. Mogelijk is de de maaktDeelUitVanPand property in plaats van alleen Pandidentififcaties. Ik maak hier een issue voor aan. Dit is niet iets dat we snel binnen deze pull request oplossen